lib/routes/devices-and-sessions.js

```diff
@@ -237,6 +243,94 @@ module.exports = (log, db, config, customs, push, devices) => {
         }
       }
     },
+    {
+      method: 'GET',
+      path: '/account/device/messages',
+      config: {
+        validate: {
+          query: {
+            service: isA.string().valid('sendtab').required(),
+            index: isA.string().optional(),
+            limit: isA.number().optional(),
```

Comment:
do you want to set a range for the `limit`?

Reply:
I don't think this is necessary

Reply:
@rfk thoughts? same in: https://github.com/mozilla/fxa-auth-server/pull/2328#discussion_r175894201

Reply:
I think we should actually enforce a maximum limit whether the client requests one or not, e.g. we'll only deliver at most 1000 messages.  Otherwise clients may not bother to implement proper paging etc (like they didn't for a long time in sync).

I'm sure @eoger will implement proper limits and paging for the initial client, but we'll eventually get one implemented by someone else :-)